docs/config-full-sample.yaml

```diff
@@ -179,6 +179,10 @@ sample:
         enabled: true
         data: []
 
+    # Private blocks publishing params
+    publishing:
+      period: 15s
```

Comment:
I know this is just a sample, but it really doesn't make sense to put publishing period less than a slot lenght even in sample.
By the way, displaying some kind of warning if publishing period is less than `slotDuration` seems like a good idea (because it never makes sense).